Automattic/wc-calypso-bridge
# Fix payments not working for business plan and move things around

This PR aims to fix the following problems:

1. WCPay menu did not show up for business plan users. This is because our frontend tooling was loaded in `class-wc-calypso-bridge.php`, which actually is only called for Ecommerce plan users.
2. Duplicated codes were made to handle business and ecom users, i.e: nav unification fixes.

This PR would bridge (🥁 ba-dum-tss) shared functionalities between business and ecom plan by sharing a single class. This class would now handle our frontend tooling registration and nav unification fixes.

Apart from that, this PR also moves WCPay-related script registration to `includes/class-wc-calypso-bridge-payments.php`, which seem to be more appropriate.

### Testing instructions

1. Load up this branch and observe everything still works regarding related features.
2. Optionally, for a better coverage please follow the tutorial in p90Yrv-2j7-p2 to test in WPCOM production. If possible, both business and ecommerce plan site is tested.
